packages/ui/v2/core/VersionInfo.tsx

```diff
@@ -0,0 +1,13 @@
+import { COMPANY_NAME, CalComVersion } from "@calcom/lib/constants";
+
+export default function VersionInfo() {
+  return (
+    <small
+      style={{
+        fontSize: "0.5rem",
```

Comment:
You can use an arbitrary value in tailwind for this; `text-[0.5rem]`. That way we don't need inline styles 😁